src/component/absensi/shift/Shift.vue

```diff
@@ -19,74 +17,93 @@
                 </tr>
             </thead>
             <tbody>
-                <tr>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-
+                <tr v-for="item in shiftItems"
+                    :key="item.id"
+                    :item="item">
+                    <td>{{item.id}}</td>
+                    <td>{{item.name}}</td>
+                    <td class="text-center">{{item.startHour}}</td>
+                    <td class="text-center">{{item.endHour}}</td>
+                    <td>{{item.department}}</td>
+                    <td>{{item.location}}</td>
                 </tr>
             </tbody>
         </table>
-
-
-    </div>
+    </main>
 </template>
 
 
 <script>
     import Alerts from '../../Alert.vue'
 
     export default {
+
         data() {
             return {
+                shiftItems: [
+                    { id: '001', name: 'Cawang Ops' , startHour:"08:00", endHour:"14:00",department:"Operation",location:"Cawang"},
+                    { id: '002', name: 'Cawang Ops' , startHour:"08:00", endHour:"16:00",department:"Operation",location:"Cawang"},
+                    { id: '003', name: 'Batu Ceper' , startHour:"07:00", endHour:"16:00",department:"Operation",location:"Batu Ceper"},
+                    { id: '004', name: 'KS Tubun BES' , startHour:"08:00", endHour:"17:00",department:"Operation",location:"location"},
+                ],
 
             };
         },
 
         components: {Alerts}
-
-
     }
 </script>
 
 <style lang="scss">
-    .container{
-        margin:0 15%;
+
+    .shift{
+        margin:3% 15%;
+        &__title{
+            display: block;
+            font-size: 1.5em;
+            font-weight: bold;
+        }
+        &__button{
+            display: inline-block;
+            padding: 8px 12px;
+            margin-bottom: 0;
+            font-size: 14px;
+            text-align: center;
+            border: 1px solid transparent;
+            border-radius: 4px;
+            background-color: #b3dcff;
+            color: black;
+            cursor: pointer;
+            text-decoration: none;
+            &:hover{
+                background-color: #92ccff;
+            }
+        }
     }
-    table{
+    .table{
         width: 100%;
-        border-collapse: collapse;
-        thead{
+        &__header{
             text-align: center;
             font-weight: bold;
         }
+        &-hover{
+            tbody{
+                tr:hover{
+                    background-color: #e1e1e1
+                }
```

Comment:
How about `shift__column:hover { ... } `? It's more BEM friendly and has more direct meaning in my opinion.